### PR TITLE
Bake EOC Free Response Solutions

### PR DESCRIPTION
**Pull request by LostMojo**
Creates an EOC Solutions section that bakes after the free response and collects the solutions into it.

### CODE REVIEW

**Comment by LostMojo** [comment]
Ready once I check w/ Brittany's copy/paste PR

File: lib/locales/en.yml [hunk 1]
```diff
@@ -16,12 +16,13 @@ en:
...
   stepwise_step_label: Step
   references: References
+  answer_key_title: Answer Key
   eoc_chapter_review: Chapter Review
   eoc_key_terms_title: Key Terms
   eoc_summary_title: Summary
   eoc_exercises_title: Exercises
   eoc_composite_metadata_title: Chapter Review
-  eoc_answer_key_title: Answer Key
+  eoc_solutions_title: Solutions
```
Comment (jpslav) [inline lib/locales/en.yml:25]: @annaludwin @wilkus27 can y'all give a polish translation for this?  

@LostMojo can you rebase to master to get the spanish translations and get @KarinaMendez2 to update them per this?

Comment (LostMojo) [reply lib/locales/en.yml:25]: Rebased and added!

**Review by KarinaMendez2 — APPROVED** [review]
looks good to me!

**Pull request merged**